packages/components/src/Table/TableCell.tsx

```diff
@@ -40,14 +40,12 @@ export const TableCell = ({ cell, align }: TableCellProps) => {
 
   const { focusProps, isFocusVisible } = useFocusRing();
   const stateProps = useStateProps({ disabled, focusVisible: isFocusVisible });
-
   return (
     <td
       ref={ref}
-      className={classNames?.cell}
+      className={cn(align ? `text-${align}` : '', classNames?.cell)}
```

Comment:
Tailwind needs to see the full CSS class here `text-right` etc. otherwhise this might break in the future :)